desc/compute/_profiles.py

```diff
@@ -753,30 +753,35 @@ def _iota_num_vacuum(params, transforms, profiles, data, **kwargs):
     description="Numerator of rotational transform formula, current contribution, "
     + "first radial derivative",
     dim=1,
-    params=["c_l"],
+    params=["c_l", "i_l"],
     transforms={"grid": []},
-    profiles=["current"],
+    profiles=["current", "iota"],
     coordinates="r",
-    data=["0", "psi_r", "psi_rr"],
+    data=["psi_r", "psi_rr", "iota_den_r", "iota_num_r vacuum"],
 )
 def _iota_num_r_current(params, transforms, profiles, data, **kwargs):
-    if profiles["current"] is None:
-        data["iota_num_r current"] = jnp.nan * data["0"]
-        return data
-
-    # 4π^2 I = 4π^2 (mu_0 current / 2π) = 2π mu_0 current
-    current = profiles["current"].compute(transforms["grid"], params["c_l"], dr=0)
-    current_r = profiles["current"].compute(transforms["grid"], params["c_l"], dr=1)
-    current_rr = profiles["current"].compute(transforms["grid"], params["c_l"], dr=2)
-    data["iota_num_r current"] = (
-        2
-        * jnp.pi
-        * mu_0
-        * transforms["grid"].replace_at_axis(
-            (current_r * data["psi_r"] - current * data["psi_rr"]) / data["psi_r"] ** 2,
-            lambda: current_rr / (2 * data["psi_rr"]),
+    if profiles["iota"] is not None:
+        iota_r = profiles["iota"].compute(transforms["grid"], params["i_l"], dr=1)
+        data["iota_num_r current"] = (
+            iota_r * data["iota_den_r"] - data["iota_num_r vacuum"]
+        )
```

Comment:
Equations 52 and 56 in the document attached to the top of GitHub PR #556 are helpful here. Recall "iota_num current" is "alpha" and "iota_num vacuum" is "beta" and "iota_den" is "gamma".
Here "iota_num_r current" is $d \alpha / d \rho$. Given $d \iota / d\rho$, we can compute that as
```python
data["iota_num_r current"] = (
    iota_r * data["iota_den"]**2
    + data["iota_num"] * data["iota_den_r"] 
) / data["iota_den"] - data["iota_num_r vacuum"]
```